packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -711,7 +711,10 @@ export const Table = <
             { [`iui-${density}`]: density !== 'default' },
             className,
           ),
-          style,
+          style: {
+            minWidth: 0, // Overrides the min-width set by the react-table but when we support horizontal scroll it is not needed
```

Comment:
Is there a point to have a test for some of these scenarios? If its not too complex

Reply:
Idk, it is pretty dumb test also since our tests are completely mocked there won't be any benefit. Maybe visual tests are better for this.